utils/integration-functions.R

```diff
@@ -0,0 +1,109 @@
+## Custom functions for data integration.
+
+# #### USAGE
+# This script is intended to be sourced in the script as follows:
+#
+# source(file.path("utils", "integration-functions.R"))
+
+plot_integration_umap <- function(sce,
+                                  integration_method,
+                                  cell_label_column,
+                                  legend_title = NULL, 
+                                  legend_labels = NULL,
+                                  plot_colors = NULL,
+                                  plot_title = NULL,
+                                  seed = NULL) {
+  
+  # Purpose: Generate UMAP plots using integration results from a SingleCellExperiment object 
```

Comment:
```suggestion
  # Purpose: Generate UMAP plots for a merged or integrated SingleCellExperiment object containing multiple batches
```
Technically this is used for unintegrated datasets, but the key is that we are plotting the batch labels so there needs to be more than one batch present.